src/data/sql-quiz.ts

```diff
@@ -1,4 +1,109 @@
+    Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
+  },
+  {
+    Question: "What are the four basic CRUD operations in SQL?",
+    Answer:
+      "The four basic CRUD operations in SQL are CREATE, READ, UPDATE, and DELETE.",
+    Distractor1: "CITE, REACH, UPDATE, DELETE",
+    Distractor2: "COPY, REJECT, UPDATE, DELETE",
+    Distractor3: "CONVERT, REVERT, UPLOAD, DELETE",
+    Explanation:
+      "CRUD stands for CREATE, READ, UPDATE, and DELETE, which are fundamental operations for managing data in a database.",
+    Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
+  },
+  {
+    Question: "What is the purpose of the SQL INSERT statement?",
+    Answer:
+      "The SQL INSERT statement is used to add new records to a table in the database.",
+    Distractor1: "It is used to modify existing records in a table.",
+    Distractor2: "It is used to delete records from a table.",
+    Distractor3: "It is used to retrieve data from a table.",
+    Explanation:
+      "The INSERT statement is used to insert new records with specified values into a table, effectively adding data to the table.",
+    Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
+  },
+  {
+    Question: "What is the purpose of the CREATE TABLE statement in SQL?",
+    Answer:
+      "The CREATE TABLE statement is used to create a new table in a database.",
+    Distractor1: "It is used to update existing table structures.",
+    Distractor2: "It is used to delete a table from the database.",
+    Distractor3: "It is used to add new records to an existing table.",
+    Explanation:
+      "The CREATE TABLE statement is used to define and create a new table in a database, specifying its columns, data types, and constraints.",
+    Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
+  },
+  {
+    Question: "What is the purpose of the SQL WHERE clause?",
+    Answer:
+      "The SQL WHERE clause is used to filter and specify the records to be retrieved, updated, or deleted based on specific conditions.",
+    Distractor1: "It is used to order records in a table.",
+    Distractor2: "It is used to define the table's primary key.",
+    Distractor3: "It is used to join multiple tables together.",
+    Explanation:
+      "The WHERE clause is used to narrow down the selection of records in a table by applying specific conditions. It is commonly used with SELECT, UPDATE, and DELETE statements to filter records.",
+    Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
+  },
+  {
+    Question: "What is SQL Injection?",
+    Answer:
+      "SQL Injection is a security vulnerability where an attacker can manipulate SQL queries by injecting malicious SQL code into input fields, potentially gaining unauthorized access to a database or damaging its content.",
```

Comment:
it is best to keep answers short
the explanation field is where you can write longer explanations 

```suggestion
      "SQL Injection is when malicious SQL code is placed into input fields",
```